pkg/ingester/ingester_v2.go

```diff
@@ -263,11 +263,16 @@ func (i *Ingester) getOrCreateTSDB(userID string) (*tsdb.DB, error) {
 
 			udir := i.userDir(userID)
 
+			var ranges []int64
+			for _, t := range i.cfg.TSDBConfig.BlockRanges {
```

Comment:
What do you think about adding a `ToMillisecondRanges() []int64` function to the data type used in the config, so that here we just do `BlockRanges: i.cfg.TSDBConfig.BlockRanges.ToMillisecondRanges()`? Makes sense?

Reply:
Makes sense